docs/UserGuide.md

```diff
@@ -218,13 +271,38 @@ Examples:
 *  `edit 1 -p 91234567 -e dbs_hr@example.com` Edits the phone number and email address of the 1st company in the currently displayed list to be `91234567` and `dbs_hr@example.com` respectively.
 *  `edit 2 -n Meta -t Engineer` Edits the name of the 2nd company to be `Meta` and change the tags to `Engineer` only.
 
-**Before:**
-![result for 'pre-editing'](images/pre_edit.png)
-
-**After:**
-![result for 'post-editing'](images/post_edit.png)
-
-### Locating companies by name: `find`
+Here are the steps to edit the fields of a company:
+1. Input: `edit 1 -n Amazon`
+   * **Before:**
+   ![result for 'pre-editing'](images/pre_edit_name.png)
+   *  **After:**
+   ![result for 'post-editing'](images/post_edit_name.png)
+2. Input: `edit 1 -e alphabet@example.com`
+   * **Before:**
+   ![result for 'pre-editing'](images/pre_edit_email.png)
+   * **After:**
+   ![result for 'post-editing'](images/post_edit_email.png)
+3. Input: `edit 1 -t UI/UX Designer`
+   * **Before:**
+   ![result for 'pre-editing'](images/pre_edit_tag.png)
+   * **After:**
+   ![result for 'post-editing'](images/post_edit_tag.png)
+4. Input: `edit 1 -p 91234567`
+    * **Before:**
+      ![result for 'pre-editing'](images/pre_edit_phone.png)
+    * **After:**
+      ![result for 'post-editing'](images/post_edit_phone.png)
+5. Input: `edit 1 -d1 2024-04-15 -d2 2024-05-01`
+    * **Before:**
+      ![result for 'pre-editing'](images/pre_edit_dates.png)
+    * **After:**
+      ![result for 'post-editing'](images/post_edit_dates.png)
+
+>:bulb: **TIP**:
+> When editing the dates of an application, it is recommended to edit both start and end dates. 
+> If only one of the date is edited, and the record is not showing anything, please refer to our [FAQ](#faq-questionquestionquestion).
+
+### Locating companies by name: `find` :mag:
```

Comment:
should we change the name of the title since now we are searching for tags as well? maybe it can be something like  Locating companies and tags by keywords